boards/msb-430/include/board.h

```diff
@@ -11,14 +11,20 @@
  * @ingroup     boards
  * @brief       Support for the ScatterWeb MSB-430 board
  *
- * <h2>Compontents</h2>
+ * @details
+ * See http://www.mi.fu-berlin.de/inf/groups/ag-tech/projects/Z_Finished_Projects/ScatterWeb/modules/mod_MSB-430.html for circuit diagram etc.
```

Comment:
Yes, the URL is long, but maybe we can still have at least one line break after it.